packages/amplify-graphql-api-construct-tests/src/__tests__/sql-models-common.ts

```diff
@@ -19,77 +21,696 @@ export const testGraphQLAPI = async (options: {
+
+  // Get Updated Todo Query
+  const getUpdatedTodo1 = await getTodo(createTodo1Result.id);
+  const getUpdatedTodo2 = await getTodo(createTodo2Result.id);
+
+  const getUpdatedTodo1Result = getUpdatedTodo1.data.getTodo;
+  const getUpdatedTodo2Result = getUpdatedTodo2.data.getTodo;
+
+  expect(getUpdatedTodo1Result.id).toEqual(createTodo1Result.id);
+  expect(getUpdatedTodo1Result.description).toEqual('Updated Todo #1');
+
+  expect(getUpdatedTodo2Result.id).toEqual(createTodo2Result.id);
+  expect(getUpdatedTodo2Result.description).toEqual('Updated Todo #2');
+
+  // List Todo Query
+  const listTodos = await listTodo();
+  const listTodosResult = listTodos.data.listTodos;
+
+  expect(listTodosResult.items.length).toEqual(2);
+  expect(listTodosResult.items).toEqual(
+    expect.arrayContaining([
+      expect.objectContaining({
+        id: getUpdatedTodo1Result.id,
+        description: 'Updated Todo #1',
+      }),
+      expect.objectContaining({
+        id: getUpdatedTodo2Result.id,
+        description: 'Updated Todo #2',
+      }),
+    ]),
   );
 
-  expect(listResult.body.data.listTodos.items.length).toEqual(1);
-  expect(todo.id).toEqual(listResult.body.data.listTodos.items[0].id);
-  const client = new LambdaClient({ region });
-  const functionName = outputs[name].SQLFunctionName;
-  const command = new GetProvisionedConcurrencyConfigCommand({
-    FunctionName: functionName,
-    Qualifier: resourceNames.sqlLambdaAliasName,
-  });
-  const response = await client.send(command);
-  expect(response.RequestedProvisionedConcurrentExecutions).toEqual(2);
+  // Delete Todo Query
+  const deleteTodo1 = await deleteTodo(createTodo1Result.id);
+  const deleteTodo1Result = deleteTodo1.data.deleteTodo;
 
-  await graphql(
-    apiEndpoint,
-    apiKey,
-    /* GraphQL */ `
-        mutation DELETE_TODO {
-          deleteTodo(input: { id: "${todo.id}" }) {
-            id
-          }
-        }
-      `,
+  expect(deleteTodo1Result.id).toEqual(createTodo1Result.id);
+  expect(deleteTodo1Result.description).toEqual('Updated Todo #1');
+
+  // List Todo Query
+  const listTodosAfterDelete = await listTodo();
+  const listTodosAfterDeleteResult = listTodosAfterDelete.data.listTodos;
+
+  expect(listTodosAfterDeleteResult.items.length).toEqual(1);
+  expect(listTodosAfterDeleteResult.items[0].id).toEqual(getUpdatedTodo2Result.id);
+  expect(listTodosAfterDeleteResult.items[0].description).toEqual('Updated Todo #2');
+
+  // Check limit and nextToken
+  const createTodo3 = await createTodo('Todo #3');
+  const createTodo4 = await createTodo('Todo #4');
+  const createTodo5 = await createTodo('Todo #5');
+
+  const createTodo3Result = createTodo3.data.createTodo;
+  const createTodo4Result = createTodo4.data.createTodo;
+  const createTodo5Result = createTodo5.data.createTodo;
+
+  expect(createTodo3Result).toBeDefined();
+  expect(createTodo3Result.id).toBeDefined();
+  expect(createTodo3Result.description).toEqual('Todo #3');
```

Comment:
as above, are you getting any meaningfully new coverage from assertions on todo3-5?

Reply:
Tests content simplified with necessary coverage on CRUDL cases.